Firestore/core/test/unit/remote/bloom_filter_test.cc

```diff
@@ -132,6 +142,131 @@ TEST(BloomFilterTest,
   }
 }
 
+class BloomFilterGoldenTest : public ::testing::Test {
+ public:
+  void RunGoldenTest(std::string& test_file) {
+    size_t start_pos = test_file.find("bloom_filter_proto");
+    if (start_pos == std::string::npos) {
+      return;
+    }
+    std::string result_file = test_file;
+    result_file.replace(start_pos, sizeof("bloom_filter_proto") - 1,
```

Comment:
Don't use `sizeof` here. Although technically it works, it's not idiomatic C++ and won't work for other types.

Instead, create a local `const std::string xxx = "bloom_filter_proto"` and use `xxx.size()` instead. Also, use this new variable in the call to `test_file.find()` above.